README.md

```diff
@@ -212,6 +212,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Jenkins](http://jenkins-ci.org/) - Provides server-based deployment services.
 * [TeamCity](http://www.jetbrains.com/teamcity/) - JetBrain's CI solution with a free version.
 * [Travis](https://travis-ci.org) - Hosted service often used for open-source projects.
+* [Concourse](https://concourse.ci/) - Provides dockerized pipeline with configuration that can be save in version control.
```

Comment:
Yes, It's language independent CI/CD tool and uses dockerized environment for build pipeline.